packages/documentation/src/content/docs/integration/deployment/docker-compose.mdx

```diff
@@ -95,26 +100,39 @@ Next update the DNS records (A records) to point to the static external IP addre
 The example domain and subdomain values are for demonstration purposes only. You must use the actual domain names that you set up with your DNS host.  
 :::
 
-## Configure compose file
+## Configure Compose file
 
-Update the variables in the following compose file with values relevant to your environment and system. Refer to the [environment variables page](/resources/environment-variables/) for details.
+The Docker Compose file is a YAML configuration file used to define the services, networks, and volumes that make up a multi-container application. In this section, we'll explore the Compose file by breaking it down into the individual Rafiki services.
 
-:::note[Update compose file]
-You must change the values enclosed within the brackets of the following compose file. Substitute `newest-version` with the latest Rafiki <LinkOut href='https://github.com/interledger/rafiki/releases'>version</LinkOut>.
+:::note[Update Compose file]
+Before using the Compose file, you must update the variables with values relevant to your environment. Specifically, change the values enclosed within the brackets and substitute `newest-version` with the latest Rafiki <LinkOut href='https://github.com/interledger/rafiki/releases'>version</LinkOut>.
 :::
 
+### Docker Compose example
```

Comment:
I think the sections here (in H4) should be available in the page's sidenav. You can do that by setting the [maxHeadingLevel](https://starlight.astro.build/guides/customization/#table-of-contents). I didn't check to see how many other H4s are in this page so we might have to get creative.